openpoiservice/server/db_import/objects.py

```diff
@@ -26,3 +30,56 @@ def __init__(self, uuid, osmid, key, value):
         self.osmid = int(osmid)
         self.key = key
         self.value = value
+
+
+class GeocoderSetup(object):
+    """Initialises geocoder"""
+
+    def __init__(self, geocoder_name):
+        self.geocoder_name = geocoder_name
+        self.geocoder = None
+
+    def define_geocoder(self):
+
+        # returns warning if no valid geocoder is provided
+        try:
+            self.geocoder_settings = get_geocoder_for_service(self.geocoder_name[0])
+
+        except Exception as err_geocoder:
+            logger.warning(err_geocoder)
+            return err_geocoder
+
+        # returns warning if no valid geocoder settings are provided
+        try:
+            if self.geocoder_name[1] is not None:
+                self.geocoder = self.geocoder_settings(**self.geocoder_name[1])
+
+            else:
+                self.geocoder = self.geocoder_settings()
+            return self.geocoder
+
+        except Exception as err_parameter:
+            logger.warning(err_parameter)
+            return err_parameter
+
+
+class AddressObject(object):
+
+    def __init__(self, lat_lng, geocoder):
+        self.lat_lng = lat_lng[::-1]
+        self.geocoder = geocoder
+
+    def address_request(self):
+
+        # address_delaied = RateLimiter(self.geocoder.reverse, min_delay_seconds=1)
+        response = self.geocoder.reverse(query=self.lat_lng)
```

Comment:
Add a try-except here to catch possible connection errors, e.g. resource not available etc.